### PR TITLE
cut 1.2.0

### PR DESCRIPTION
**Pull request by gbraad**
Preparation for release

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # Various versions - Minishift, default OpenShift, default B2D ISO
-MINISHIFT_VERSION = 1.1.0
+MINISHIFT_VERSION = 1.2.0
```
Comment (praveenkumar) [inline Makefile]: LGTM 👍

Comment (gbraad) [inline Makefile]: @LalatenduMohanty suggested to also integrate @budhrg's PR. At the moment I am waiting for this...

When submitted as PR, we also have a build and integration triggered to confirm this build is OK.

Comment (hferentschik) [inline Makefile]: I suggest to go ahead with the release

**Comment by gbraad** [comment]
https://www.redhat.com/archives/container-tools/2017-June/msg00006.html

**Pull request merged**